luq/rbf_fit.py

```diff
@@ -355,7 +363,7 @@ def initialize_params(self,
         # initialize sigmas
```

Comment:
@taylor-s-roper Please add a description of param output_data_shifted here.

Reply:
done